bin/seed-golfathlete-db.js

```diff
@@ -0,0 +1,70 @@
+#! /usr/bin/env node
+
+'use strict'
+
+const async = require('async')
+const Athlete = require('../src/models/athlete')
+
+const mongoose = require('mongoose')
+mongoose.set('useCreateIndex', true)
+mongoose.connect('mongodb://localhost:27017/golfAthleteTest', { useNewUrlParser: true })
+mongoose.Promise = global.Promise
+const db = mongoose.connection
+db.on('error', console.error.bind(console, 'MongoDB connection error:'))
+
+const athletes = []
+
+function createAthlete (lastName, firstName, ghinNumber, cachedGhinIndex, cachedGhinIndexDate, driverClubHeadSpeed, dob, address, description, cb) {
+  const now = Date.now()
+
+  const athlete = new Athlete({
```

Comment:
is there a reason not to use object shorthand?

Reply:
1. My preference is actually to use async/await whenever possible. Since Express, Mongoose, MongoDB is new technology for me, I follow the some examples as-is and have gone back to update the code yet. I do feel it's important to have consistency in how Promises are used and do like the improved readability of async/await.
2. _id/id? Oversight on my part. I actually plan to pull out the model into its own file and will change _id to be id. This way the model would be more portable since it wouldn't have any MongoDB-ism in it.

Reply:
3. Regarding the structuring of the test, I was focused on getting the tests in place and working. I agree I need to group the tests more logically by describe -> it, it, it... etc. so they are easier to follow.
4. Tomasz pointed out using the spred operator also. I was not familiar with it. However, now that I am, I will definitely use it because I like the conciseness of it.

Reply:
P.S. To answer why I did not use id in the first place, it was because id was not mapping correctly to MongoDB's _id in the database. I still need to look into this a bit more.